types/custom-functions-runtime/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "ban-types": false
```

Comment:
What does ban-types: false do to make index.d.ts' compilation pass? [Palantir's documentation](https://palantir.github.io/tslint/rules/ban-types/) only explains what it does with a parameter like `[true, ["Object", "Use {} instead", "String"]`

If possible, I'd like to see this rule removed too.

Reply:
Is it possible to use `(...args: any[]) => any` instead? This type has the advantage that it works better with advanced types (`ReturnType<T>` for example), but it means that users can't store functions created via `Function('code')` in CustomFunctionMappings. 

The test example uses a normal function declaration `ADD10`. Do you know what users will do?

Reply:
Added a note on the ADD10 so it is hopefully more clear. There will also be additional documentation on this on Microsoft's docs site, so it should be understandable now. I will check with some devs who are more knowledgeable than I on the Typescript change.

Reply:
Thanks for your feedback.  I consulted with our dev team and while they agree in principle, for this particular case, they think it's clearer to just type it as Function.  Essentially, the CustomFunctionMappings object is just a redirecter to the right function; and the d.ts is also used for auto-generated reference docs, where "Function" looks cleaner than the (...args: any[]) => any notation (esp. for non-TypeScript developers).  So if it's OK with you, we'd like to submit as is for now, and we can tweak in the future if we see a clear advantage for this scenario to be typed differently